tests/test_commands.py

```diff
@@ -954,6 +954,17 @@ def test_unlink_with_multiple_keys(self, r):
         assert r.get("a") is None
         assert r.get("b") is None
 
+    @pytest.mark.onlynoncluster
+    # @skip_if_server_version_lt("7.0.0") turn on after redis 7 release
+    def test_lcs(self, unstable_r):
+        unstable_r.mset({"foo": "ohmytext", "bar": "mynewtext"})
+        assert unstable_r.lcs("foo", "bar") == b"mytext"
```

Comment:
Does this not decode when decode_responses is True? If not we have a bug.

Reply:
It decodes, I test it like that because decode_response=False in unstable_r